x-pack/plugins/cases/server/files/mime_types.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * These were retrieved from https://www.iana.org/assignments/media-types/media-types.xhtml#image
+ */
+const imageMimeTypes = [
```

Comment:
The grouping of types in this file is prob the best candidate for the select box in the attachments table.

Kinda like what @cnasikas suggested
<img width="200" alt="Screenshot 2023-02-28 at 9 56 08 AM" src="https://user-images.githubusercontent.com/1533137/221862969-9c580f2c-d746-4520-882b-55ce94cc2546.png">